### PR TITLE
feat(Gmail(52 > main)) link connection with a service

### LINKED ISSUE #52: Link conneciton with a service
Create a button to connect to google service and create the table

### PR DESCRIPTION
**Pull request by Pablodeibar**
add a button on the front to connect to OAuth 2.0 for google
Closes #52

### CODE REVIEW

**Review by AlexGuillard — APPROVED** [review]
ok

File: server/src/auth/auth.service.ts [hunk 1]
```diff
@@ -135,6 +135,8 @@ export class AuthService {
         };
       }
 
+      console.log(existingUser);
```
Comment (Benjicatch) [inline server/src/auth/auth.service.ts:138]: remove

**Review by Benjicatch — CHANGES_REQUESTED** [review]

**Review by FoxaxeWasTaken — APPROVED** [review]
LGTM !

**Review by AlexGuillard — DISMISSED** [review]
ok

**Review by FoxaxeWasTaken — APPROVED** [review]
LGTM !

**Review by Benjicatch — APPROVED** [review]

**Pull request merged**